examples/material_gallery/lib/demo/buttons_demo.dart

```diff
@@ -3,214 +3,267 @@
 // found in the LICENSE file.
 
 import 'package:flutter/material.dart';
-import 'package:flutter/widgets.dart';
 
-import 'tabs_fab_demo.dart';
-import 'dialog_demo.dart';
-import 'snack_bar_demo.dart';
-
-const String _floatingText =
-  "A floating action button is a circular material button that lifts "
-  "and displays an ink reaction on press. It turns and fades in when "
-  "it changes.";
+import '../gallery/demo.dart';
 
 const String _raisedText =
-  "A raised button is typically a rectangular material button that lifts "
-  "and displays ink reactions on press. Raised buttons add dimension to "
-  "mostly flat layouts. They emphasize functions on busy or wide spaces.";
+  "# Raised buttons\n"
+  "Raised buttons add dimension to mostly flat layouts. They emphasize"
+  "functions on busy or wide spaces.";
+
+const String _raisedCode =
+"""// Create a new flat button
+new RaisedButton(
+  child: new Text('BUTTON TITLE'),
+  onPressed: () {
+    // Perform some action
+  }
+);
+
+// Create disabled button
+new RaisedButton(
+  child: new Text('BUTTON TITLE')
+);""";
 
 const String _flatText =
-  "A flat button is made of ink that displays ink reactions on press "
-  "but does not lift. Use flat buttons on toolbars, in dialogs and "
-  "inline with padding";
+  "# Flat buttons\n"
+  "Flat buttons are printed in material. They do not lift, but fill with "
+  "color on press.\n\n"
+  "Use flat buttons in the following locations:\n\n"
+  "- On toolbars\n"
+  "- In dialogs, to unify the button action with the dialog content\n"
+  "- Inline, with padding, so the user can easily find them";
 
-const String _dropdownText =
-  "A dropdown button selects between multiple selections. The button "
-  "displays the current state and a down arrow.";
-
-class _ButtonDemo {
-  _ButtonDemo({ this.title, this.text, this.builder }) {
-    assert(title != null);
-    assert(text != null);
-    assert(builder != null);
+const String _flatCode =
+"""// Create a new flat button
+new FlatButton(
+  child: new Text('BUTTON TITLE'),
+  onPressed: () {
+    // Perform some action
   }
+);
 
-  final String title;
-  final String text;
-  final WidgetBuilder builder;
+// Create disabled button
+new FlatButton(
+  child: new Text('BUTTON TITLE')
+);""";
 
-  TabLabel get tabLabel => new TabLabel(text: title.toUpperCase());
+const String _dropdownText =
+  "# Dropdown buttons\n"
+  "A dropdown button selects between multiple selections. The button displays "
+  "the current state and a down arrow. Available states may be shown as a "
+  "list of strings, a palette, or icons, for example.";
```

Comment:
Note also: since we're referring to the dropdown's selections as "values" in the code snippet, I think it would be best to do so here also.  So here's another version with some extra text thrown in at no extra cost:

A dropdown button displays a menu that's used to select a value from a small set of values. The button displays the current value and a down arrow.